src/main/scala/com/atomist/rug/kind/core/ProjectMutableView.scala

```diff
@@ -428,14 +429,23 @@ class ProjectMutableView(
           case wtf =>
             throw new RugRuntimeException(ed.name, s"Unexpected editor failure: $wtf", null)
         }
-        case _ => throw new MissingRugException(s"Unable to find editor: [$editorName]")
+        case None =>
+          if(editorName.contains(":")){
+            val shortName = StringUtils.substringAfterLast(editorName, ":")
+            if(rug.findRug(shortName).nonEmpty){
+              throw new EditorNotFoundException(s"Could not find editor: $editorName. Did you mean: $shortName?")
```

Comment:
Could we perhaps indicate that fully-qualified names are for editors from other Rugs while short names are only for editors in this project. This would lead the user towards a better understanding of when to use one vs the other.